### PR TITLE
Generate correct mr for NIST FIPS and NBS FIPS

### LINKED ISSUE #47: (URGENT) FIPS PUBS MR scheme wrong (`NIST.FIPS` not `NIST.FIPS.PUB`)
See https://github.com/metanorma/nist-pubid/blob/main/README.adoc
Screenshot 2022-01-10 at 3 33 04 PM

See these:

MR | DOI
-- | --
NBS.FIPS.PUB.34 | NBS.FIPS.34
NBS.FIPS.PUB.35 | NBS.FIPS.35
NBS.FIPS.PUB.36 | NBS.FIPS.36
NBS.FIPS.PUB.36-1975 | NBS.FIPS.36-1975

### PR DESCRIPTION
**Pull request by mico**
closes #47

### CODE REVIEW

**Comment by ronaldtse** [comment]
@mico tests are currently failing.

**Comment by ronaldtse** [comment]
@mico the correct MR form is `LC nnnn` instead of `LCIRC nnnn` for both NBS and NIST.

https://nvlpubs.nist.gov/nistpubs/Legacy/LC/nbslettercircular1127.pdf

Screenshot 2022-01-11 at 12 40 55 PM

https://nvlpubs.nist.gov/nistpubs/Legacy/LC/nistlettercircular1136.pdf
Screenshot 2022-01-11 at 12 41 10 PM

File: spec/nist_pubid/nist_tech_pubs_spec.rb [hunk 1]
```diff
@@ -83,7 +83,7 @@
   describe "#status" do
     let(:id) { "LCIRC 897" }
     let(:doi) { "NBS.LCIRC.897" }
-    let(:mr) { "NBS.LC.897" }
+    let(:mr) { "NBS.LCIRC.897" }
```
Comment (ronaldtse) [inline spec/nist_pubid/nist_tech_pubs_spec.rb:86]: `LC` not `LCIRC`.

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: spec/nist_pubid/nist_tech_pubs_spec.rb (hunk 1 above)
Comment (mico) [reply spec/nist_pubid/nist_tech_pubs_spec.rb:86]: image

Comment (mico) [reply spec/nist_pubid/nist_tech_pubs_spec.rb:86]: Should we update https://github.com/metanorma/nist-pubid/blob/main/README.adoc as well?

Comment (ronaldtse) [reply spec/nist_pubid/nist_tech_pubs_spec.rb:86]: Yes, please help do that. Thanks!

**Review by ronaldtse — APPROVED** [review]
Thanks @mico !

**Pull request merged**